src/dbt_score/evaluation.py

```diff
@@ -57,11 +57,11 @@ def evaluate(self) -> None:
             self.results[model] = {}
             for rule in rules:
                 try:
-                    result: RuleViolation | None = rule.evaluate(model, **rule.config)
+                    if rule.should_evaluate(model):  #  Consider model filter(s).
```

Comment:
IIUC, if a filter skips the model, it's not now visible to the user. In the case of complex filters, it might be important to flag skipping so the filter behaves expectedly 

It might be interesting to either:
* emit a warning if the model skipped (maybe too much noise)
* Keep track of skipped models in the skipped list attr. In the end, `Formatter` can use this to print at least the number of skipped models.

My proposal is to add `self.skips` and track Model, Filter

I am fine to outline it as a new feature request.

Reply:
Yes let's keep it for another PR, then we can already release this one :rocket: